src/hevm/test/rpc.hs

```diff
@@ -43,11 +54,94 @@ tests = testGroup "rpc"
                                                    )
 
         assertEqual "coinbase" (Addr 0x690b9a9e9aa1c9db991c7721a92d351db4fac990) cb
-        assertEqual "number" (Fetch.BlockNumber numb) block'
+        assertEqual "number" (BlockNumber numb) block'
         assertEqual "basefee" 22163046690 basefee
         assertEqual "prevRan" 0x2267531ab030ed32fd5f2ef51f81427332d0becbd74fe7f4cd5684ddf4b287e0 prevRan
     ]
+  , testGroup "execution with remote state"
+    -- execute against remote state from a ds-test harness
+    [ testCase "dapp-test" $ do
+        let testFile = "test/contracts/pass/rpc.sol"
+        runDappTestCustom testFile ".*" Nothing False testRpcInfo >>= assertEqual "test result" True
+
+    -- concretely exec "transfer" on WETH9 using remote rpc
```

Comment:
Perhaps we can reference `https://etherscan.io/token/0xc02aaa39b223fe8d0a0e5c4f27ead9083c756cc2#code` here somewhere?

Reply:
done.